### DISCUSSION TITLE
DOOM + DOOM II ships with DOSBox Staging

### DISCUSSION THREAD

**Discussion by Grounded0** [dosbox-staging/dosbox-staging#3909] [General]
DOOM + DOOM II ships with DOSBox Staging:

https://steamdb.info/depot/2281/

https://store.steampowered.com/app/2280/DOOM__DOOM_II/

https://www.youtube.com/watch?v=IlPSbp1io7w&t=229s

**Comment by johnnovak** [comment]
Nice. Would be a lot nicer if they showed us some money too 😎 💸 🤑

**Comment by Grounded0** [comment]
> Nice. Would be a lot nicer if they showed us some money too 😎 💸 🤑

I'm staring at you, Bethesda.

**Reply by Torinde** [reply]
Can they be contacted? Or it's not considered appropriate?

**Comment by johnnovak** [comment]
Merged this into https://github.com/dosbox-staging/dosbox-staging/discussions/3833

- https://github.com/dosbox-staging/dosbox-staging/discussions/3833
